base/applications/regedit/lang/pt-PT.rc

```diff
@@ -15,53 +15,53 @@ END
 /* Command-line strings */
 STRINGTABLE
 BEGIN
-    STRING_USAGE "Usage:\n\
-  regedit [options] [filename] [reg_key]\n\n\
-Options:\n\
-  [no option]    Launch the graphical version of this program.\n\
-  /L:system.dat  The location of the system.dat file to be modified.\n\
-                 Compatible with any other switch. Ignored.\n\
-  /R:user.dat    The location of the user.dat file to be modified.\n\
-                 Compatible with any other switch. Ignored.\n\
-  /C             Import the contents of a registry file.\n\
-  /D             Delete a specified registry key.\n\
-  /E             Export the contents of a specified registry key to a file.\n\
-                 If no key is specified, the entire registry is exported.\n\
-  /S             Silent mode. No messages will be displayed.\n\
-  /V             Launch the GUI in advanced mode. Ignored.\n\
-  /?             Display this information and exit.\n\
-  [filename]     The location of the file containing registry information to\n\
-                 be imported. When used with [/E], this option specifies the\n\
-                 file location where registry information will be exported.\n\
-  [reg_key]      The registry key to be modified.\n\
+    STRING_USAGE "Uso:\n\
+  regedit [Opções] [Ficheiro] [Chave]\n\n\
+Opções:\n\
+  [sem opções]    Iniciar a versão gráfica deste programa.\n\
```

Comment:
The last commit introduced this glitch, which was pushed after my initial review.
```suggestion
  [sem opções]   Iniciar a versão gráfica deste programa.\n\
```